internal/git/git_test.go

```
@@ -0,0 +1,209 @@
+	// Initialize git repository
+	cmd := exec.Command("git", "init")
+	cmd.Dir = tempDir
+	if err := cmd.Run(); err != nil {
+		os.RemoveAll(tempDir)
+		t.Fatalf("Failed to init git repo: %v", err)
+	}
+
+	// Set git user info for commits
+	cmd = exec.Command("git", "config", "user.name", "Test User")
+	cmd.Dir = tempDir
+	cmd.Run()
```

golangci-lint (1.64.8)

Error return value of `cmd.Run` is not checked

(errcheck)